src/main/java/Baymax.java

```diff
@@ -0,0 +1,59 @@
+import java.util.Scanner;
+
+public class Baymax {
+    public static void main(String[] args) {
+        String name = "Baymax";
+        System.out.println("Hello! I'm " + name + ", your personal task manager." + System.lineSeparator() + "What can I do for you today?" + System.lineSeparator() + System.lineSeparator());
+
+        Task[] list = new Task[100];
+        int num = 0;
+        String text;
+
+        while(true) {
+
+            Scanner in = new Scanner(System.in);
+            text = in.nextLine();
+
+            // "bye" -- terminate
+            if (text.equalsIgnoreCase("bye")) {
+                break;
+            }
+
+            // "mark" -- mark task
+            else if (text.startsWith("mark")) {
+                int position = Integer.parseInt(text.substring(5)) - 1;
```

Comment:
Perhaps you can rename "position" to something that is clearer - what position is it referring to?